fluentui_transients/src/main/java/com/microsoft/fluentui/tooltip/Tooltip.kt

```diff
@@ -132,7 +132,7 @@ class Tooltip {
 
         // Get location of anchor view on screen
         val screenPos = IntArray(2)
-        anchor.getLocationOnScreen(screenPos)
+        anchor.getLocationInWindow(screenPos)
```

Comment:
Why do we need this? All calculations are done as per screen right?
Also, how will showAtlocation behave if we are getting location with window reference?

Reply:
Both of these return the same values in most conditions. This changes helps in solving shifts(due to status bar) when device is rotated 90 ACW.
 I noticed that at 90 ACW, tooltip was getting shifted to the right by the height of the status bar.